src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    public static final String SEPARATE = System.lineSeparator();
+    public static final String SPLITTER = ",";
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
 
+    public String readFile(String fileName) {
```

Comment:
All additional methods must be private
All private methods should be below public methods